datanode/gateway/graphql/schema.graphql

```diff
@@ -2157,6 +2167,12 @@ type Position {
   "Average entry price for this position"
   averageEntryPrice: String!
 
+  "The total amount of profit and loss that was not transferred due to loss socialisation"
+  lossSocializationAmount: String!
+
+  "Enum set if the position was closed out or orders were removed because party was distressed"
+  positionStatus: PositionStatus!
```

Comment:
suggestion (non blocking): would prefer this as just `status` the code will read nicer as a result: `position.status` rather than `position.positionStatus`